app/views/users/_user.haml

```diff
@@ -5,7 +5,7 @@
     - unless current_user == user
       #follow_form
       - if current_user.following? user
```

Comment:
Think of a more elegant approach to this. Note that the only difference between line#8 and line#10 is the partial file